### PR TITLE
#32: Implement Roll API for Dice So Nice support.

### PR DESCRIPTION
**Pull request by mattraykowski**

### CODE REVIEW

**Comment by mattraykowski** [comment]
@Armaklan please verify that I did not accidentally break your recent roll improvements.

File: module/actor/sheets/base.js [hunk 1]
```diff
@@ -889,6 +887,7 @@ export default class ActorSheetSS2e extends ActorSheet {
     template,
     title,
   }) {
+    console.log('rolldata', rolldata);
```
Comment (Armaklan) [inline module/actor/sheets/base.js:890]: To delete ?

File: module/actor/sheets/base.js [hunk 2]
```diff
@@ -965,39 +964,34 @@ export default class ActorSheetSS2e extends ActorSheet {
...
       parseInt(form.trait.value) +
       parseInt(form.bonusDice.value);
-    let incThreshold = 0;
-    if (form.increaseThreshold !== undefined)
-      incThreshold = form.increaseThreshold.checked;
+
+    const incThreshold =
+      form.increaseThreshold !== undefined ? form.increaseThreshold.checked : 0;
 
     let addOneToDice = 0;
```
Comment (Armaklan) [inline module/actor/sheets/base.js:971]: Add ternary expression in addOneToDice too

**Review by Armaklan — APPROVED** [review]

**Comment by Armaklan** [comment]
> 

I'll test this branch this evening after my game :)

**Comment by Armaklan** [comment]
Test and is ok :)

**Pull request merged**